bosco-cluster-remote-hosts.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
-# Source condor-ce environment
-[ -f /usr/share/condor-ce/condor_ce_env_bootstrap ] &&
-   . /usr/share/condor-ce/condor_ce_env_bootstrap
+. foreach_bosco_endpoint.sh
```

Comment:
I forget the semantics of source... doesn't that source `foreach_bosco_endpoint.sh` from the cwd? Perhaps it should be `. /usr/local/bin/foreach_bosco_endpoint.sh`?

Reply:
`PATH` is searched, too, which i presume contains `/usr/local/bin`, but if we wanna make it explicit i guess that's fine too.

For your reference:

```
$ help source
source: source filename [arguments]
    Execute commands from a file in the current shell.
    
    Read and execute commands from FILENAME in the current shell.  The
    entries in $PATH are used to find the directory containing FILENAME.
    If any ARGUMENTS are supplied, they become the positional parameters
    when FILENAME is executed.
    
    Exit Status:
    Returns the status of the last command executed in FILENAME; fails if
    FILENAME cannot be read.
```